### DISCUSSION TITLE
cron job

### DISCUSSION THREAD

**Discussion by leedsrhinos** [boxbilling/boxbilling#1227] [General]
hi guys need help with the cron job
i set the cron up in cpanel dont i??
if so is this correct
image below
![cron](https://user-images.githubusercontent.com/100451638/158678689-5003b443-fbf5-4299-91d3-36bf9697224b.jpg)

**Comment by givinghawk** [comment]
Yea that is correct

**Reply by leedsrhinos** [reply]
thanks but when i go to image below
![cron 2](https://user-images.githubusercontent.com/100451638/158679705-70d1f767-2029-49ba-928c-817095536afc.jpg)

it was executed 7 mins ago when i manualy pressed the aeroplane execute should it not execute every 5 mins?
if so it was 7 mins since i did it manualy 
Thanks

**Comment by leedsrhinos** [comment]
Anyone ? It's now showing 16 h since it was executed?
